src/AsyncWriterResult/AsyncWriterResultOp.fs

```diff
@@ -0,0 +1,15 @@
+namespace AsyncWriterResult.Operator.AsyncWriterResult
+
+open AsyncWriterResult
+
+/// Operators for working with the AsyncWriterResult type
+[<AutoOpen>]
+module AsyncWriterResult =
+    /// AsyncWriterResult.map
+    let (<!>) f x = AsyncWriterResult.map f x
+
+    /// AsyncWriterResult.apply
+    let (<*>) f x = AsyncWriterResult.apply f x
+
+    /// AsyncWriterResult.bind
+    let (>>=) x f = AsyncWriterResult.bind f x
```

Comment:
I thought the value of moving the operators into a separate file/module was to give the client the option to `open`. So didn't expect we'd `[<AutoOpen>]` everywhere.

logaf low, but is this done to maintain parity with the existing version and minimise effort after upgrading, or are we following FsT?

Reply:
This is still following FsT - initially I was thinking it had to be this way modules and namespaces work across multiple files but as far as I can tell, there aren't any operators across multiple files.

In this case, we have

```
namespace AsyncWriterResult.Operator.AsyncWriterResult
[<AutoOpen>]
module AsyncWriterResult =
```
which means you can open `AsyncWriterResult.Operator.AsyncWriterResult` which will give you access to the operators, but I don't see how this is different in practice to
```
namespace AsyncWriterResult.Operator
module AsyncWriterResult =
```
🤷